app/views/partials/_menu.html.haml

```diff
@@ -0,0 +1,11 @@
+%h1 Today's Menu
+- @categories.each do |category|
+    %h2= category.category
+    %div{id: dom_id(category)}
+    - category.products.each do |product|
+        %h4= product.name
+        %p= product.description
+        %p= product.price
+        - if user_signed_in?  
+            - if current_user.owner
+                = link_to "Edit #{category.category}", edit_product_path(product.id)
```

Comment:
Not sure if I understand. The link is `"Edit #{category.category}"` but you link to `edit_product_path`. Shouldn't it be a path to edit a category? Or is the link value wrong?

Reply:
We are using string interpolation to add 3 edit buttons (Edit Starter,Edit Main and Edit Dessert) (As shown in screenshot 1), depending on eash link selection, that particular product id will be used to edit the product information. Hence we are using category to uniquely identify the link, and selecting product based on that